src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java

```diff
@@ -0,0 +1,60 @@
+package ca.bc.gov.open.jag.efiling.demo;
+
+import ca.bc.gov.open.jag.efilingcommons.model.AccountDetails;
+import ca.bc.gov.open.jag.efilingcommons.model.CreateAccountRequest;
+import ca.bc.gov.open.jag.efilingcommons.service.EfilingAccountService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SeedAccount {
+
+    private static final Logger logger = LoggerFactory.getLogger(SeedAccount.class);
+
+    private final EfilingAccountService efilingAccountService;
+
+    public SeedAccount(EfilingAccountService efilingAccountService){
+
+        this.efilingAccountService = efilingAccountService;
+
+    }
+
+    @EventListener
+    public void onApplicationEvent(ContextRefreshedEvent event) {
+        init();
+    }
+
+    private void init() {
+
+        CreateAccountRequest createAccount1 = CreateAccountRequest
+                .builder()
+                .universalId(Keys.ACCOUNT_WITH_EFILING_ROLE)
+                .firstName("temp")
```

Comment:
Temp??? No we can do better 🤣🤣🤣

Reply:
Im with Adam, one should be Bob Ross and the other should be Steve or Steven Ross